backend/repository/data/client/contacts_client_update.json

```diff
@@ -4,7 +4,7 @@
     "type": "http",
     "thriftFile": "clients/contacts/contacts.thrift",
     "exposedMethods": {
-        "SaveContacts": "Contacts::saveContacts"
+        "saveContacts": "Contacts::saveContacts"
```

Comment:
what's the rationale making the method name lowercase? The generated method will always have first letter uppercase.